### PR TITLE
Added Computer Graphics Resource

### LINKED ISSUE #1225: 💡 [FEATURE] - <title> Adding Resource for Computer Graphics
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

_No response_

### Explain your solution

_No response_

### Any alternative approaches/features

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by SadafKausar2025**
# Pull Request  #1225 

## Description
Please include a summary of the changes and the related issue.  

Closes #1225 

## Type of change
- [x] Adding a new Resource
- [ ] Adding a new Category
- [ ] Documentation enhancement or fixes
- [ ] Website enhancement or fixes

## Checklist:
- [x] I read carefully [CONTRIBUTING.md](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Screenshots (if appropriate):

Please add screenshots to help explain your changes.

### CODE REVIEW

**Comment by SadafKausar2025** [comment]
hello @jfmartinz 
can you please review this PR?

File: Computer Graphics/readme.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+| Resource                                                       | Description                                                                                                                                                           |
+| :------------------------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [CGSociety](http://cgsociety.org/)                             | CGSociety is a community for digital artists and professionals working in the computer graphics industry, offering forums, galleries, and articles on various topics. |
+| [Polycount](https://polycount.com/)                            | Polycount is a community for game artists focused on 3D art, modeling, texturing, and game design, with forums, challenges, and resources for artists.                |
+| [Real-Time Rendering Community](http://realtimerendering.com/) | The Real-Time Rendering Community offers resources and discussions on real-time rendering techniques, tools, and technologies.                                        |
```
Comment (jfmartinz) [inline Computer Graphics/readme.md:224]: Please take a look at these format. You should use the 'table' tags.

**Review by jfmartinz — CHANGES_REQUESTED** [review]
You also remove the readme for this category
image

**Comment by SadafKausar2025** [comment]
> You also remove the readme for this category image

yes this was previously done by me . and you said anyone else is doing this so , I removed it

**Comment by SadafKausar2025** [comment]
hello @jfmartinz 
please see is this alright?
the NoSql code was of mine and you said that anyone else is doing work so I removed it

**Comment by jfmartinz** [comment]
> hello @jfmartinz please see is this alright? the NoSql code was of mine and you said that anyone else is doing work so I removed it

My bad. Yes removed it.

File: Computer Graphics/readme.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
+# Computer Graphics
+
+Computer graphics refers to the imathat are generated using a computer. This can include 2D images, such as photos or illustrations, as well as 3D images, which are generated using specialized software and require more processing power. Computer graphics are used in a wide range of applications, including video games, animation, architecture, engineering, and medical imaging. Advances in computer technology have made it possible to create increasingly realistic and detailed graphics, and the use of computer graphics continues to grow and evolve.
+
+## Table of Contents
+
+- [Roadmap](#Roadmap)<br>
+- [Tutorials or Courses](#tutorials-or-courses)<br>
```
Comment (jfmartinz) [inline Computer Graphics/readme.md:8]:
```suggestion
- [Courses](#courses)<br>
```

File: Computer Graphics/readme.md [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+<table width="100%" id="Roadmap">
+      <tr>
+        <th>Resource Name</th>
+        <th>Description</th>
+      </tr>
+      <th ><a href="https://github.com/prographon/graphics-developer-roadmap">Computer Graphics </a></th>
+      <th>This is the Roadmap for Graphics Engineer</th>
+</table>
+
+## Tutorials or Courses
```
Comment (jfmartinz) [inline Computer Graphics/readme.md:37]:
```suggestion
## Courses
```

File: Computer Graphics/readme.md [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+            <td></td>
+        </tr>
+        <tr>
+            <td>Tomas Akenine-Möller</td>
+            <td><a href="https://www.realtimerendering.com/">Real-Time Rendering 4th Edition</a></td>
+            <td>Advanced</td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
```
Comment (jfmartinz) [inline Computer Graphics/readme.md:246]:
```suggestion

```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Once these request changes is resolved, ill merged it. Thanks

**Comment by SadafKausar2025** [comment]
hello @jfmartinz 
please review the changes.
Is it alright?

**Comment by SadafKausar2025** [comment]
Hello @jfmartinz 
Please merge my pr

**Review by jfmartinz — APPROVED** [review]

**Pull request merged**